OpenRA.Mods.Common/Traits/World/WarheadDebugOverlay.cs

```diff
@@ -0,0 +1,85 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Drawing;
+using OpenRA.Graphics;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Part of the combat overlay from DeveloperMode. Attach this to the world actor.")]
+	public class WarheadDebugOverlayInfo : ITraitInfo
+	{
+		public readonly int DisplayDuration = 20;
```

Comment:
Up this to a full second please (maybe even longer, but nukes and such already take too long due to their 50 warheads).